src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -364,6 +377,16 @@ type WebAppConfig =
                   Tags = this.Tags}
             | _ ->
                 ()
+
+            yield! this.SiteExtensions
+                |> List.map (fun x ->
+                    {
```

Comment:
Style: Records should follow style guide [here](https://github.com/CompositionalIT/farmer/blob/master/CONTRIBUTING.md#5-adhere-to-coding-standards)

Reply:
I'll check that style guide, thanks.

I'm going to take this opportunity to suggest that we refactor the `buildResources` function. It's extremely hard to orient yourself in a function that lasts for several screen lengths.

Reply:
As it's an interface method I'm not sure what we can do with it really, but you can certainly break out the implementation into separate functions. Happy to review this with you.